js/src/widgets/searchWithinResults.js

```diff
@@ -437,14 +437,12 @@ $.SearchWithinResults.prototype = {
             '{{canvaslabel}}',
           '</a>',
           '{{#if annotations}}',
-            '<div>',
```

Comment:
Ok, looks quite weird for me, I have put the 'div' back in the searchWithinResults.js template block, to pass the test.